functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -50,6 +46,7 @@ func Test5LatencyTesting(t *testing.T) {
 	if err != nil {
 		data, _ := json.Marshal(ds) // we can safely skip errors
 		testlog.Infof("DaemonSet %s/%s image=%q status:\n%s", ds.Namespace, ds.Name, images.Test(), string(data))
+		deleteNamespace()
```

Comment:
In general the ```Test5LatencyTesting``` function should be as minimal as possible and just call the entry point for ginkgo test:
```
func Test5LatencyTesting(t *testing.T) {
	RegisterFailHandler(Fail)
	rr := []Reporter{}
	if ginkgo_reporters.Polarion.Run {
		rr = append(rr, &ginkgo_reporters.Polarion)
	}
	rr = append(rr, junit.NewJUnitReporter("latency_testing"))
	RunSpecsWithDefaultAndCustomReporters(t, "Performance Addon Operator latency tools testing", rr)
```

Then you can have all the extra logic, pre-pulling the image, declaring the client, etc. on a ```BeforeSuite()```
Then even if the test will fail, the ```AfterSuite()``` will get called anyway